### PR TITLE
Remove non-working links

### LINKED ISSUE #11705: Non working links
codebetter.com has been down for some time.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 84d83855-4370-90e8-024a-12c5ac8220e9
* Version Independent ID: 01be3db1-4a00-c8f2-aa59-59656f206b93
* Content: [Applying CQRS and CQS approaches in a DDD microservice in eShopOnContainers](https://docs.microsoft.com/en-us/dotnet/standard/microservices-architecture/microservice-ddd-cqrs-patterns/eshoponcontainers-cqrs-ddd-microservice#feedback)
* Content Source: [docs/standard/microservices-architecture/microservice-ddd-cqrs-patterns/eshoponcontainers-cqrs-ddd-microservice.md](https://github.com/dotnet/docs/blob/master/docs/standard/microservices-architecture/microservice-ddd-cqrs-patterns/eshoponcontainers-cqrs-ddd-microservice.md)
* Product: **dotnet**
* Technology: **dotnet-ebooks**
* GitHub Login: @CESARDELATORRE
* Microsoft Alias: **wiwagn**

### PR DESCRIPTION
**Pull request by mvelosop**
## Summary

There was no replacement links after contacting the blog author.

Closes https://github.com/dotnet/docs/issues/11705

### CODE REVIEW

**Comment by mairaw** [comment]
There are two other topics linking to that site: https://github.com/dotnet/docs/search?utf8=%E2%9C%93&q=codebetter&type= 

Do you wanna tackle them on the same PR?

**Comment by mvelosop** [comment]
Done @mairaw!

File: docs/standard/microservices-architecture/microservice-ddd-cqrs-patterns/apply-simplified-microservice-cqrs-ddd-patterns.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ CQRS is an architectural pattern that separates the models for reading and writi
...
+[Command and Query Responsibility Segregation (CQRS)](https://martinfowler.com/bliki/CQRS.html) was introduced by Greg Young and strongly promoted by Udi Dahan and others. It is based on the CQS principle, although it is more detailed. It can be considered a pattern based on commands and events plus optionally on asynchronous messages. In many cases, CQRS is related to more advanced scenarios, like having a different physical database for reads (queries) than for writes (updates). Moreover, a more evolved CQRS system might implement [Event-Sourcing (ES) (page 25)](https://cqrs.files.wordpress.com/2010/11/cqrs_documents.pdf#page=25) for your updates database, so you would only store events in the domain model instead of storing the current-state data. However, this is not the approach used in this guide; we are using the simplest CQRS approach, which consists of just separating the queries from the commands.
```
Comment (mairaw) [inline docs/standard/microservices-architecture/microservice-ddd-cqrs-patterns/apply-simplified-microservice-cqrs-ddd-patterns.md]: This link doesn't seem very applicable. Isn't there something more approachable we can link to?
Perhaps https://martinfowler.com/eaaDev/EventSourcing.html?

File: docs/standard/microservices-architecture/microservice-ddd-cqrs-patterns/apply-simplified-microservice-cqrs-ddd-patterns.md [hunk 2]
```diff
@@ -29,6 +29,11 @@ An example of this kind of service is the ordering microservice from the eShopOn
 
 The application layer can be the Web API itself. The important design aspect here is that the microservice has split the queries and ViewModels (data models especially created for the client applications) from the commands, domain model, and transactions following the CQRS pattern. This approach keeps the queries independent from restrictions and constraints coming from DDD patterns that only make sense for transactions and updates, as explained in later sections.
 
+## Additional resources
+
+ - **Greg Young. Versioning in an Event Sourced System** (Free online e-book) \
```
Comment (mairaw) [inline docs/standard/microservices-architecture/microservice-ddd-cqrs-patterns/apply-simplified-microservice-cqrs-ddd-patterns.md]: I see a minimum price of $5? I don't think we should be linking to paid resources,

**Review by mairaw — CHANGES_REQUESTED** [review]
Changes look good but I left a few questions/comments.

File: docs/standard/microservices-architecture/microservice-ddd-cqrs-patterns/apply-simplified-microservice-cqrs-ddd-patterns.md (hunk 1 above)
Comment (mvelosop) [reply docs/standard/microservices-architecture/microservice-ddd-cqrs-patterns/apply-simplified-microservice-cqrs-ddd-patterns.md]: Yep, will update as per your suggestion.

File: docs/standard/microservices-architecture/microservice-ddd-cqrs-patterns/apply-simplified-microservice-cqrs-ddd-patterns.md (hunk 2 above)
Comment (mvelosop) [reply docs/standard/microservices-architecture/microservice-ddd-cqrs-patterns/apply-simplified-microservice-cqrs-ddd-patterns.md]: The book is free to read online, anyway I checked with @CESARDELATORRE first.

image

Perhaps changing it to "Free to read online e-book"?

Comment (mairaw) [reply docs/standard/microservices-architecture/microservice-ddd-cqrs-patterns/apply-simplified-microservice-cqrs-ddd-patterns.md]: It's easy to miss that. How about you keep the Free online e-book but link to https://leanpub.com/esversioning/read instead?

Comment (mvelosop) [reply docs/standard/microservices-architecture/microservice-ddd-cqrs-patterns/apply-simplified-microservice-cqrs-ddd-patterns.md]: Yep, certainly much better 😊

**Review by mairaw — APPROVED** [review]
Thanks @mvelosop and sorry for the delay.

**Comment by mvelosop** [comment]
No worries 😊

**Pull request merged**